src/variables/utils/validation.ts

```diff
@@ -7,6 +7,16 @@ import {
 
 const reservedVarNames = [TIME_RANGE_START, TIME_RANGE_STOP, WINDOW_PERIOD]
 
+export const fullErrorText =
+  'Variable name must start with a letter or underscore, and ' +
+  'contain only numbers, letters, and underscores.'
+
+export const emptyErrorText = 'Variable name cannot be empty'
+export const uniqError = 'Variable name must be unique'
```

Comment:
Nitpick here. Constants should be in all caps. Something like:
`FULL_ERROR_TEXT`
`EMPTY_ERROR_TEXT`
`UNIQUE_ERROR_TEXT`